docs/utilities/tags.md

```diff
@@ -5,7 +5,7 @@ Tags are generalized sets of objects in the game used for grouping related thing
 
 Declaring Your Own Groupings
 ----------------------------
-Tags are declared in your mod's [datapack][datapack]. For example, `/data/<modid>/tags/blocks/foo/tagname.json` will declare a `Tag$Named<Block>` with ID `modid:foo/tagname`.
+Tags are declared in your mod's [datapack][datapack]. For example, `/data/<modid>/tags/blocks/foo/tagname.json` will declare a `TagKey<Block>` with ID `modid:foo/tagname`.
```

Comment:
Perhaps it is more accurate to reverse this definition: that a `TagKey` of a given location leads to a certain file location in the datapack.

Also, we may want to add a section describing the names of the tag folders, especially mentioning the special case of the IIRC five previous registries with tags in previous versions retaining their tags folder in the plural rather than following the registry's name.